package.json

```diff
@@ -160,5 +161,7 @@
   "peerDependencies": {
     "amphora-search": "^6.2.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "nouislider": "^12.0.0"
```

Comment:
this makes me sad!!!!!!!!

Reply:
wait why

Reply:
(nouislider is a really good, pretty lightweight library that implements a lot of functionality that is just completely unsupported with native html sliders)

Reply:
oh it doesn't need to be a regular dep, right. good catch!!!

Reply:
i liked having 0 deps!!!!!!!